package.json

```diff
@@ -93,7 +93,6 @@
     "@typescript-eslint/eslint-plugin": "^1.13.0",
     "@typescript-eslint/parser": "^1.13.0",
     "babel-plugin-closure-elimination": "^1.3.0",
-    "babel-plugin-strip-invariant": "^1.0.0",
     "babel-plugin-transform-async-to-promises": "^0.8.14",
     "babel-plugin-transform-dev-warning": "^0.1.1",
```

Comment:
Shall we also replace this plugin with your transformer?

Reply:
Added it :)